syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.ev.fap.person";

import "person.proto";

package person;

service PersonService {
  rpc SearchPerson(SearchPersonRequest) returns (SearchPersonResponse) {}
}

message SearchPersonRequest {
  string name_contains = 1;
  int32 min_age = 2;
  int32 max_age = 3;
}

message SearchPersonResponse {
  bool success = 1;
  string message = 2;
  repeated Person persons = 3;
}